app/views/Licenses.js

```diff
@@ -76,7 +76,7 @@ export const LicensesScreen = ({ navigation }) => {
             use='headline2'
             onPress={() => Linking.openURL(PRIVACY_POLICY_URL)}
             style={styles.privacyPolicy}>
-            {t('label.privacy_policy')}
+            {t('label.terms_of_use')}
```

Comment:
not sure this is reflected in en.json?

Reply:
hm. the key is used on the `Legal` screen to link to the privacy site. Probably does make more sense to just rename the key rather than blindly update the value.

Reply:
renamed.